csrc/scheduler/pointwise_utils.h

```diff
@@ -63,11 +70,12 @@ class DomainMap {
 
 // Returns number of non-reduction/non-broadcas/non-device dims in logical
 // domain
-inline int64_t nRootDims(const TensorView* tv) {
+inline int64_t nLogicalDims(const TensorView* tv) {
   auto logical_dom = tv->getLogicalDomain();
   int64_t tv_n_dims = 0;
   for (auto dim : logical_dom) {
-    if (!dim->isReduction() && !dim->isBroadcast() && !dim->isDeviceDim()) {
+    if (!dim->isReduction() && !dim->isBroadcast() &&
+        !dim->hasExpandedExtent() && !dim->isDeviceDim()) {
```

Comment:
Do we need to check `dim->hasExpandedExtent()` since we already checked `dim->isBroadcast()`?

Reply:
sorry, another unintended changes. Let me revert this one.